Formula/dafny.rb

```diff
@@ -5,6 +5,11 @@ class Dafny < Formula
   sha256 "ea7ae310282c922772a46a9a85e2b4213043283038b74d012047b5294687d168"
   revision 2
 
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+b?)$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

We should be using the standard regex for Git tags like `1.2.3`/`v1.2.3`, as there's no need to have the optional `b?` for this repository.

Reply:
totally!